en/docs/security-announcements/security-advisories/2019/WSO2-2019-0655.md

```diff
@@ -27,12 +27,12 @@ Several JavaScript libraries used by the XACML entitlement user interfaces of WS
 
 
 ### IMPACT
-A malicious entity may intercept the unencrypted HTTP request used to retrieve the JavaScript content and/or alter the unencrypted HTTP response to include malicious content, in combination with other attack vectors such as man-in-the-middle attacks.
+A malicious entity may intercept the unencrypted HTTP request used to retrieve the JavaScript content and/or alter the unencrypted HTTP response to include malicious content, in combination with other attack vectors such as Eavesdropper attacks.
```

Comment:
Shall we use the `man-in-the-middle` term here.

Reply:
This term change was part of a discussion we did around using gender terms. +1 to the new suggestion.